src/shared/meson.build

```diff
@@ -131,7 +131,7 @@ shared_sources = files(
         'pkcs11-util.c',
         'pretty-print.c',
         'ptyfwd.c',
-        'pwquality-util.c',
+        'pwquality-dlopen.c',
```

Comment:
Maybe, `password-quality-util-pwquality.c` is better?
Note that we have firewall-util-iptables.c and firewall-util-nft.c.